src/lib/server/models.ts

```diff
@@ -66,6 +67,7 @@ const modelConfig = z.object({
 		.optional(),
 	multimodal: z.boolean().default(false),
 	unlisted: z.boolean().default(false),
+	embeddingModelName: validateEmbeddingModelByName(embeddingModels).optional(),
```

Comment:
```suggestion
	embeddingModel: validateEmbeddingModelByName(embeddingModels).optional(),
```

I think this field can be just renmaed to `embeddingModel`. In this case, you'd need to update the references that were using `embeddingModelName` before

Reply:
here as well https://github.com/huggingface/chat-ui/blob/feature/assistants/src/routes/conversation/%2Bserver.ts#L40-L41